build/tutorials/platform/integrate-exchange-with-avalanche.md

```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
 
 Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
 
+## Integration using Rosetta
+
+[Rosetta](https://www.rosetta-api.org/) is an open-source specification and set of tools that makes integrating with different blockchain networks easier by presenting the same set of APIs for every network. The Rosetta API is made up of 2 core components, the Data API and the Construction API. Together, these APIs allow for anyone to read and write to blockchains in a standard format over a standard communication protocol. The specifications for these APIs can be found in the rosetta-specifications repository.
```

Comment:
Let's fill in the links for Data API, Construction API, and rosetta-specifications (which can be found in this paragraph: https://www.rosetta-api.org/docs/welcome.html#start-building).